From eabcccfd371343d1f464f89e6be65faddb1464c3 Mon Sep 17 00:00:00 2001
From: Keith Packard <keithp@keithp.com>
Date: Thu, 18 Jan 2024 12:25:03 -0800
Subject: [PATCH 01/12] Ignore document errors during build

---
 gcc/genhooks.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gcc/genhooks.cc b/gcc/genhooks.cc
index b984bee69..e0da55d56 100644
--- a/gcc/genhooks.cc
+++ b/gcc/genhooks.cc
@@ -145,8 +145,8 @@ emit_documentation (const char *in_fname)
 	    fatal ("Duplicate hook %s\n", sh.name);
 	  shp->pos = i;
 	}
-      else
-	fatal ("No place specified to document hook %s\n", sh.name);
+//      else
+//	fatal ("No place specified to document hook %s\n", sh.name);
       free (sh.name);
     }
   /* Copy input file to stdout, substituting @hook directives with the
-- 
2.51.0

